gravitee-apim-portal-webui-next/src/components/nav-bar/nav-bar-button/nav-bar-button.component.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { CommonModule } from '@angular/common';
+import { Component, Input } from '@angular/core';
+import { MatButton } from '@angular/material/button';
+import { RouterModule } from '@angular/router';
+
+@Component({
+  selector: 'app-nav-bar-button',
+  standalone: true,
+  imports: [RouterModule, MatButton, CommonModule],
+  templateUrl: './nav-bar-button.component.html',
+})
+export class NavBarButtonComponent {
+  @Input()
+  path: string[] = [''];
```

Comment:
Just wondering, why it has to be an array instead of just plain string?

Reply:
The `[routerLink]` takes a `string[]`. The path could potentially be `['api', 'my-favorite-api']`.

Reply:
Okay